mk/citrix.list

```diff
@@ -1,2 +1,2 @@
-deb     http://updates.vmd.citrix.com/@PRODUCT_BRAND@/@PRODUCT_VERSION@/debian/ ^DISTRO^ main
```

Comment:
is the `xensource` ready now?

Reply:
This feature is not yet implemented and needs to be discussed. I have to confirm whether citrix.list and citrix.repo still need to exist. If not I can remove these two.